ruby/vampires.rb

```diff
@@ -0,0 +1,95 @@
+puts "What is your name?"
+user_name = gets.chomp.to_s
+
+puts "How old are you?"
+age = gets.chomp.to_i 
+
+puts "What year were you born?"
+year = gets.chomp.to_i
+
+current_year = 2017
+verified_age = current_year - year
+
+puts "Would you care for some garlic bread? (yes or no)"
+bread = gets.chomp.downcase
+
+until bread == "yes" || bread == "no"
+	puts "Would you care for some garlic bread? (yes or no)"
+	bread = gets.chomp.downcase
+end
+
+puts "Would you like to enroll in our insurance plan? (yes or no)"
+insurance = gets.chomp.downcase
+
+until insurance == "yes" || insurance == "no"
+	puts "Would you like to enroll in our insurance plan? (yes or no)"
+	insurance = gets.chomp.downcase
+end
+
+puts "Please list the items you are allergic to one at a time. (say done when finished)"
+allergy = gets.chomp.downcase
+
+until allergy == "done" || allergy == "sunshine"
+	allergy = gets.chomp.downcase
+end
+
+if (user_name != "Drake Cula" && user_name != "Tu Fang") && (age == verified_age || age == verified_age - 1) && (bread == "yes" || insurance == "yes") && (allergy != "sunshine") 
```

Comment:
Consider nesting your control flow.  In this case if the we know the employee's name is Drac of Fang then we don't need to run anymore of these conditions. 

```
if name == "Drake Cula" || name == "Tu Fang"
  results = "Definitely a vampire"
else
  # run other checks
end
```